### PR TITLE
Request Reparse

### PR DESCRIPTION
**Pull request by dragon-dxw**
## Trello card / Rollbar error (etc)
https://trello.com/c/HMXLmwmk/1485-add-ability-for-editors-to-manually-trigger-a-document-reparse-proof-of-ability-to-reparse

- [ ] Requires env variable(s) to be updated: `REPARSE_SNS_TOPIC`
- [ ] Consider changing `SNS_TOPIC` to `ANNOUNCE_SNS_TOPIC`

### CODE REVIEW

File: tests/factories.py [hunk 1]
```diff
@@ -14,8 +14,9 @@ class DocumentFactory:
         "uri": ("uri", "test/2023/123"),
         "name": ("name", "Judgment v Judgement"),
         "neutral_citation": ("neutral_citation", "[2023] Test 123"),
+        "best_human_identifier": ("neutral_citation", "[2023] Test 123"),
         "court": ("court", "Court of Testing"),
-        "judgment_date_as_string": ("judgment_date_as_string", "2023-02-03"),
+        "document_date_as_string": ("document_date_as_string", "2023-02-03"),
```
Comment (jacksonj04) [inline tests/factories.py:19]: 🎉

**Review by jacksonj04 — COMMENTED** [review]
A few quick comments. Substance all looks good to me though!

File: src/caselawclient/models/utilities/aws.py [hunk 2]
```diff
@@ -189,3 +195,40 @@ def build_new_key(old_key: str, new_uri: str) -> str:
...
+            "s3Bucket": env("PRIVATE_ASSET_BUCKET"),
+            "s3Key": generate_docx_key(uri),
+            "reference": reference or f"FCL-{uuid.uuid4()}",
+            "originator": "FCL",
+            "parserInstructions": parser_instructions,
+        },
+    }
+
+    client.publish(
+        TopicArn=env("REPARSE_SNS_TOPIC"),
```
[the remark this answers is not in the document]
Comment (dragon-dxw) [reply src/caselawclient/models/utilities/aws.py:231]: This is how we've been told to do it, and it seems to work. We can change this later if asked.

File: setup.cfg [hunk 3]
```diff
@@ -17,11 +17,11 @@ where = src
 caselawclient = xquery/*.xqy
 
 [flake8]
-max-line-length = 120
```
[the remark this answers is not in the document]
Comment (dragon-dxw) [reply setup.cfg:20]: We're ignoring this, we'll probably ruff it up at some point.

**Review by jacksonj04 — APPROVED** [review]

**Pull request merged**